packages/x-charts/src/ChartsLegend/ChartsLegend.tsx

```diff
@@ -506,6 +506,7 @@ ChartsLegend.propTypes = {
   direction: PropTypes.oneOf(['column', 'row']),
   /**
    * Set to true to hide the legend.
+   * @default false
```

Comment:
is this intended?

Reply:
Clearly: https://app.circleci.com/pipelines/github/mui/mui-x/46768/workflows/eb3685d4-f5af-4772-8205-8c02fe582754/jobs/268408 🙈